Various/ak5k_Low latency monitoring.lua

```diff
@@ -1,7 +1,18 @@
 -- @description Low latency monitoring
 -- @author ak5k
--- @version 1.7
--- @changelog Bug fix related to track validation, which could cause script to crash when changing projects or deleting tracks while script is enabled.
+-- @version 0.2.1
```

Comment:
The new version number is lower than the previous one (v1.7 > v0.2.1). This won't trigger an update in ReaPack. Would using 2.1 instead be OK?

Reply:
I forgot one 'debug check' in. Maybe I could close this pull request and re-upload with cleaned code and version v2.1?